security/guard_authentication.rst

```diff
@@ -412,11 +412,18 @@ Each authenticator needs the following methods:
     :class:`Symfony\\Component\\HttpFoundation\\Response` object that helps
     the user authenticate (e.g. a 401 response that says "token is missing!").
 
-**supportsRememberMe**
+**supportsRememberMe()**
     If you want to support "remember me" functionality, return true from this method.
     You will still need to active ``remember_me`` under your firewall for it to work.
     Since this is a stateless API, you do not want to support "remember me"
     functionality in this example.
+    
+**createAuthenticatedToken(UserInterface $user, string $providerKey)**
+    If you are implementing the :class:`Symfony\\Component\\Security\\Guard\\GuardAuthenticatorInterface`
+    class instead of extending the :class:`Symfony\\Component\\Security\\Guard\\AbstractGuardAuthenticator`
```

Comment:
@powerdan Sorry about that but class that you added here should be reverted. `GuardAuthenticatorInterface` is an interface, but not a class. Though as `Interface` is already part of the name we shouldn't add the word "interface" instead of "class". We should only keep the one after `AbstractGuardAuthenticator`.